lambdas/service/app.py

```diff
@@ -902,6 +905,11 @@ def filter_schema(relation: str) -> JSON:
 
 
 def repository_search_params_spec():
+    # This spec is shared across all entity types, so no single value would be
+    # correct here. Using the largest possible value ensures that we don't
+    # incorrectly reject valid requests, at the cost of accepting some invalid
+    # ones.
```

Comment:
Don't understand this comment. The specs aren't used for validation, only documentation. You can specify here whatever you want without affecting what's accepted and what's rejected.

If you have to pick a value, use the value that works everywhere.

Also consider not specifying a maximum value at all.

You could also enforce a minimal maximum pages size i.e. a page size that must be supported in every catalog and for every entity type.

Reply:
The Swagger UI uses the values specified in the spec to do some client side validation.

![swagger-validation](https://github.com/DataBiosphere/azul/assets/46407464/66be2b0d-d7dc-46c1-a291-7c6f3323d9cc)

The maximum can be omitted.

![Screen Shot 2023-10-09 at 4 59 30 PM](https://github.com/DataBiosphere/azul/assets/46407464/57a5c970-1f02-4868-8a86-0ef4461cf962)